diff --git a/build/components/Openfort/OpenfortProvider.js b/build/components/Openfort/OpenfortProvider.js
index 4372b476f50988a316e88eef70c39286f2357bd8..135c5b145552a90c8b48c33b12d5a928e2026ccd 100644
--- a/build/components/Openfort/OpenfortProvider.js
+++ b/build/components/Openfort/OpenfortProvider.js
@@ -12,11 +12,11 @@ import { isFamily } from '../../utils/wallets.js';
 import { OpenfortContext, UIContext } from './context.js';
 import { UIAuthProvider, routes, defaultSendFormState, defaultBuyFormState, notStoredInHistoryRoutes } from './types.js';
 
-const SolanaContextProvider = lazy(() => import(/* @vite-ignore */ '../../solana/SolanaContext.js').then((m) => ({ default: m.SolanaContextProvider })));
-const LazyEmbeddedWalletWagmiSync = lazy(() => import(/* @vite-ignore */ '../../wagmi/useEmbeddedWalletWagmiSync.js').then((m) => ({
+const SolanaContextProvider = lazy(() => import('../../solana/SolanaContext.js').then((m) => ({ default: m.SolanaContextProvider })));
+const LazyEmbeddedWalletWagmiSync = lazy(() => import('../../wagmi/useEmbeddedWalletWagmiSync.js').then((m) => ({
     default: m.EmbeddedWalletWagmiSync,
 })));
-const LazyConnectKitModal = lazy(() => import(/* @vite-ignore */ '../ConnectModal/index.js'));
+const LazyConnectKitModal = lazy(() => import('../ConnectModal/index.js'));
 let openfortProviderWarnedNoWagmi = false;
 /**
  * Root provider for Openfort. Wrap your app with this to enable connect modal, auth, and wallet features.
